pkg/3scale/amp/operator/system_options_provider.go

```diff
@@ -557,6 +563,11 @@ func (s *SystemOptionsProvider) sidekiqPodTemplateLabels() map[string]string {
 	for k, v := range s.commonSidekiqLabels() {
 		labels[k] = v
 	}
+	if s.apimanager.Spec.System.SidekiqSpec.Labels != nil {
```

Comment:
this `if` is not needed

Reply:
Addressed in this [commit](https://github.com/3scale/3scale-operator/pull/835/commits/f903bf41523372ad2297594e4efac295a12c0665)